examples/typescript/src/App.tsx

```diff
@@ -15,36 +17,36 @@ interface PostData {
 }
 
 const client = new GraphQLClient({
-  url: 'https://create-react-app-server-kqtv5azt3q-ew.a.run.app'
+  url: 'http://localhost:8080/graphql'
```

Comment:
why this change? I think the idea is that we deploy this as a demo app, IIRC. you will see that there is definitely a graphql server running at that url. I appreciate that this is something that you may want to run locally when you're developing, so let's maybe take the chance to make this dynamic

Reply:
There seems to be a mix across the examples. Some use a local GQL server and others connect to this demo. I can look in to making it dynamic if required though

Reply:
we have a demo app running, let's primarily make sure that that one keeps running

Reply:
👍  Reverted the original value